app/controllers/application_controller/ci_processing.rb

```diff
@@ -44,11 +44,8 @@ def set_ownership(klass = "VmOrTemplate")
       @edit[:explorer] = true
       ownership
     else
-      render :update do |page|
-        page << javascript_prologue
-        if role_allows(:feature => "vm_ownership")
-          page.redirect_to :controller => "#{rec_cls}", :action => 'ownership'              # redirect to build the ownership screen
-        end
+      if role_allows(:feature => "vm_ownership")
+        javascript_redirect :controller => "#{rec_cls}", :action => 'ownership' # redirect to build the ownership screen
```

Comment:
The situation should not happen unless you are trying to run stuff that you don't have permission for. But you are right --> `adding render :nothing`.